plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -1290,16 +1284,19 @@ const string read_only::KEYi64 = "i64";
 read_only::get_info_results read_only::get_info(const read_only::get_info_params&, const fc::time_point&) const {
    const auto& rm = db.get_resource_limits_manager();
 
-   auto head_id = db.head().id();
-   auto lib_id = db.last_irreversible_block_id();
-   auto fhead_id = db.fork_db_head().id();
+   auto head = db.head();
+   auto head_id = head.id();
+   auto fork_root = db.fork_db_root();
+   auto fork_head = db.fork_db_head();
```

Comment:
Now this seems inconsistent. Why do we have both `fork_root` and `fork_db_root` at different spots in the code?

Reply:
I don't really like `fork_root` because it sounds like is the the `root` of a `fork`, not of `fork_db`. If we are going to spell it out, I think it should be `fork_db_root` everywhere, although I did prefer the shorter `froot` :-).